elements/lisk-transactions/src/12_multisignature_transaction.ts

```diff
@@ -124,7 +118,7 @@ export class MultisignatureTransaction extends BaseTransaction {
 		}
 
 		// Check if empty signatures are present
-		if (this.signatures.includes('')) {
+		if (!this.signatures.length) {
```

Comment:
i think this logic is different? it needs to check if each signature has some value i think.
`!this.signatures.every(signature => signature.length > 0)`